certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -557,6 +568,60 @@ PROCEDURE GETSHAREDETAILS
       END
     END
   END
+
+ AUTOTRANSFERLISTCOUNT=0
+ IF SCURRENTSTATUS="Renew" THEN
+  DO
+   FOR ACCOUNT TARGETACCOUNT
+    DO
+     SHAREFOUND=FALSE
+     FOR EACH SHARE WITH (SHARE:ID=TARGETSHAREID)
+      DO
+       TARGETMATURITYDATE=SHARE:MATURITYDATE
+       SHAREFOUND=TRUE
+      END
+     UNTIL SHAREFOUND=TRUE 
+  
+     FOR EACH SHARE WITH (SHARE:CLOSEDATE=DATENULL AND
+                          SHARE:ID<>TARGETSHAREID)
+                           
+      DO
+       SHARETRANSFERFOUND=FALSE
+       TRANSFERAMTTOTAL=$0.00
+       TMPCHR="0000"+SHARE:ID
+       TMPCHR=SEGMENT(TMPCHR,LENGTH(TMPCHR)-(IDLENGTH-1),LENGTH(TMPCHR))
+       TMPCHR=TARGETACCOUNT+"S"+TMPCHR
+
+       FOR EACH SHARE TRANSFER WITH ((SHARE TRANSFER:EXPIRATIONDATE=DATENULL OR
+                                     SHARE TRANSFER:EXPIRATIONDATE>SYSTEMDATE) AND
+                                     SHARE TRANSFER:TYPE=3 AND
+                                     SHARE TRANSFER:ACCOUNTNUMBER=TARGETACCOUNT AND
+                                     SHARE TRANSFER:IDTYPE=0 AND
+                                     SHARE TRANSFER:AMOUNT<>$0.00 AND
+                                     SHARE TRANSFER:EFFECTIVEDATE=TARGETMATURITYDATE-1 AND
+                                     SHARE TRANSFER:NEXTDATE=TARGETMATURITYDATE-1 AND
+                                     SHARE TRANSFER:FREQUENCY=0)
+        DO
+         SHARETRANSFERFOUND=TRUE
+         TRANSFERAMTTOTAL=TRANSFERAMTTOTAL+SHARE TRANSFER:AMOUNT
+        END
+
+       IF SHARETRANSFERFOUND=TRUE THEN
+        DO
+         AUTOTRANSFERLISTCOUNT=AUTOTRANSFERLISTCOUNT+1
```

Comment:
Should we be evaluating the incrementation of AUTOTRANSFERLISTCOUNT against a max value? Perhaps TRANSFERLISTMAX = 99 (as well as in existing code (line 513 +/-)

Reply:
Added this code.